### DISCUSSION TITLE
请问树莓派怎么通过源码编译安装？

### DISCUSSION THREAD

**Discussion by zhangyiming748** [SagerNet/SagerNet#38] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**
我现在有 树莓派3和4，请问怎么在，任意一个设备上编译安装这个工具？

**Comment by nekohasekai** [comment, marked as answer]
这是 Android 应用，不能安装。
